libmamba/src/core/transaction_context.cpp

```diff
@@ -180,13 +179,8 @@ namespace mamba
         m_pyc_process = std::make_unique<reproc::process>();
 
         reproc::options options;
+#ifndef _WIN32
```

Comment:
we might be able to not do this with static micromamba builds. We can add a variable or something like that later.